KartographischeAktion/aktionskarten-frontend
# Add continuous integration support (travis)

* check code quality wih jscs (`npm run syntax`)
* `npm run` scripts a defined in package.json

To test for yourself run the following:

```
$ npm install
$ npm run syntax
```

Functionality tests are still missing. See as well #12.
